easybuild/easyconfigs/c/CUDA/CUDA-10.0.130.eb

```diff
@@ -11,8 +9,8 @@ description = """CUDA (formerly Compute Unified Device Architecture) is a parall
 
 toolchain = SYSTEM
 
-source_urls = ['https://developer.nvidia.com/compute/%(namelower)s/%(version_major_minor)s/Prod/local_installers/']
-sources = ['%(namelower)s_%(version)s_410.48_linux']
+source_urls = ['https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/']
+sources = ['cuda_%(version)s_410.48_linux']
```

Comment:
@casparvl Any specific reason for these changes?

Reply:
@boegel No. I guessI initially added this EasyConfig to the PR because my local EasyBuild installation didn't have an official CUDA (?). No matter: I'll just remove it from the PR anyway, since we're now basing the build on CUDA-9.2.88.